src/config.rs

```diff
@@ -485,8 +485,8 @@ impl Config {
             },
             bindings: Keybindings {
                 global: GlobalKeybindings {
-                    scroll_down: keybinding!([KeyCode::Char('j');]),
-                    scroll_up: keybinding!([KeyCode::Char('k');]),
+                    scroll_down: keybinding!([KeyCode::Char('j');, KeyCode::Down;]),
```

Comment:
Could you please also add a line to the `CHANGELOG` mentioning these new defaults? (Since this is the first change after 0.9.1, just create a new `# Unreleased v0.9.2-pre` section)

Reply:
Also please modify the keybinding documentation [here](https://github.com/Builditluc/wiki-tui/blob/main/docs/docs/configuration/keybindings.md?plain=1#L77)